cocos/spine/lib/spine-core.d.ts

```diff
@@ -760,7 +760,7 @@ declare namespace spine {
     enum TextureFilter {
         Nearest = 9728,
         Linear = 9729,
-        MipMap = 9987,
+        MipMap = 9930,
         MipMapNearestNearest = 9984,
         MipMapLinearNearest = 9985,
         MipMapNearestLinear = 9986,
```

Comment:
https://github.com/EsotericSoftware/spine-runtimes/blob/b09e189c6438cb9358005771c3304b8f52bc4e30/spine-ts/spine-core/src/Texture.ts#L49C5-L49C5
```ts
export enum TextureFilter {
	Nearest = 9728, // WebGLRenderingContext.NEAREST
	Linear = 9729, // WebGLRenderingContext.LINEAR
	MipMap = 9987, // WebGLRenderingContext.LINEAR_MIPMAP_LINEAR
	MipMapNearestNearest = 9984, // WebGLRenderingContext.NEAREST_MIPMAP_NEAREST
	MipMapLinearNearest = 9985, // WebGLRenderingContext.LINEAR_MIPMAP_NEAREST
	MipMapNearestLinear = 9986, // WebGLRenderingContext.NEAREST_MIPMAP_LINEAR
	MipMapLinearLinear = 9987 // WebGLRenderingContext.LINEAR_MIPMAP_LINEAR
}
```

Why change the MipMap value, it's wrong. The configuration of eslint which ensure the enum value should be different is too strict.

Reply:
The values of MipMap and MipMapLinearLinear are the same. I have only changed the wrong parts to other values. If you know the correct value, you should change it to the value you know.